src/common/scripts/modules/Block.ts

```diff
@@ -0,0 +1,166 @@
+import EventBus from './EventBus';
+
+export default class Block {
+    static EVENTS = {
+        INIT: 'init',
+        FLOW_CDM: 'flow:component-did-mount',
+        FLOW_CDU: 'flow:component-did-update',
+        FLOW_RENDER: 'flow:render'
+    };
+   
+    _element: HTMLElement = document.createElement('template');
+    _meta: {
+        tagName: string
+        props: Record<string, any>
```

Comment:
Если используешь в проекте `;`, то старайся использовать везде
```suggestion
        tagName: string;
        props: Record<string, any>;
```